src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,541 @@
+/// \brief Detailed implementation of Bjorhus-type boundary corrections
+namespace Bjorhus {
+/// @{
+/*!
+ * \brief Computes the expression needed to set boundary conditions on the time
+ * derivative of the characteristic field \f$v^{0}_{iab}\f$
+ *
+ * \details In the Bjorhus scheme, the time derivatives of evolved variables are
+ * characteristic projected. Constraint-preserving correction terms
+ * \f$T^{\mathrm C}_{ab}\f$, Sommerfeld condition terms for gauge degrees of
+ * freedom \f$T^{\mathrm G}_{ab}\f$, and terms constraining physical degrees of
+ * freedom \f$T^{\mathrm P}_{ab}\f$ are added here to the resulting
+ * characteristic (time-derivative) field:
+ *
+ * \f{align}
+ * \Delta \partial_t v^{-}_{ab} = T^{\mathrm C}_{ab} + T^{\mathrm G}_{ab}
+ *                              + T^{\mathrm P}_{ab}.
+ * \f}
+ *
+ * These terms are given by the following expressions from
+ * \cite Lindblom2005qh :
+ *
+ * \f{eqnarray}
+ * T^{\mathrm C}_{ab} &=& \frac{1}{4}
+ *      \left(k^c k^d l_a l_b - k^c l_b P^d_a - k^c l_a P^d_b - k^d l_b P^c_a
+ *            - k^d l_a P^c_b + 2 P^{cd} P_{ab}\right) \partial_t v^{-}_{ab}\\
+ *     &&+ \frac{1}{2} \lambda_{-}c^{\hat{0}-}_c \left(\frac{1}{2} l_a l_b k^c +
+ *          P_{ab} l^c - P^c_b l_a - P^c_a l_b \right) \nonumber
+ * \f}
+ *
+ * where \f$l^a (l_a)\f$ is the outgoing null vector (one-form), \f$k^a (k_a)\f$
+ * is the incoming null vector (one-form), \f$P_{ab}, P^{ab}, P^a_b\f$ are
+ * spacetime projection operators defined in `transverse_projection_operator()`,
+ * \f$\partial_t v^{-}_{ab}\f$ is the characteristic projected time derivative
+ * of evolved variables (corresponding to the \f$v^{-}\f$ field), and
+ * \f$c^{\hat{0}\pm}_a\f$ are characteristic modes of the constraint evolution
+ * system:
+ *
+ * \f{align}\nonumber
+ * c^{\hat{0}\pm}_a = F_a \mp n^k C_{ka},
+ * \f}
+ *
+ * where \f$F_a\f$ is the generalized-harmonic (GH) F constraint [Eq. (43) of
+ * \cite Lindblom2005qh], \f$C_{ka}\f$ is the GH 2-index constraint [Eq. (44)
+ * of \cite Lindblom2005qh], and \f$n^k\f$ is the unit spatial normal to the
+ * outer boundary.
+ *
+ * \f{align}
+ * T^{\mathrm G}_{ab} = \frac{1}{2}
+ *     \left(k_a P^c_b l^d + k_b P^c_a l_d - \frac{1}{2}
+ *          \left(k_a l_b k^c l^d + k_b l^a k_c l^d + k_a k_b l^c l^d
+ *          \right) \right) \left(\gamma_2 - \frac{1}{r}
+ *                          \right) \partial_t v^{\psi}_{ab}
+ * \f}
+ *
+ * where \f$r\f$ is the radial coordinate at the outer boundary, which is
+ * assumed to be spherical, \f$\gamma_2\f$ is a GH constraint damping parameter,
+ * and \f$\partial_t v^{\psi}_{ab}\f$ is the characteristic projected time
+ * derivative of evolved variables (corresponding to the \f$v^{\psi}\f$ field).
+ * Finally,
+ *
+ * \f{align}
+ * T^{\mathrm P}_{ab} = \left( P^c_a P^d_b - \frac{1}{2} P_{ab} P^{cd} \right)
+ *     \left(\partial_t v^{-}_{cd} + \lambda_{-}
+ *           \left(U^{3-}_{cd} - n^i C_{icd}\right)\right),
```

Comment:
I think the way it is currently coded, this should have a factor of `\gamma_2`.

Reply:
yes, added.